making flocking and gliding more open (#337)
CHTC/chtc-website-source: uw-research-computing/scaling-htc.md

================================================

Running on other resources in addition to CHTC has one huge benefit:
size! The UW Grid and Open Science Pool (OS Pool) include thousands of computers,

> addition to what\'s already available in CHTC, including specialized 

Author: ```suggestion
in addition to what\'s already available in CHTC, including specialized 
```

<a name="osg"></a>

> B. OSG Pool

Reviewer: ```suggestion
B. UW-Madison's OSG Pool
```

------------------------------

CHTC maintains an OSG pool for the campus community, which includes 
resources contributed by campuses, national labs, and other institutions 
across and beyond the US.

> We call sending jobs to other institutions in our OSG pool *gliding*.

Reviewer: ```suggestion
When you send jobs to other institutions in our OSG pool, we call that *gliding*.
```

The <a href="http://www.osg-htc.org/">OSG Consortium</a> is a group of universities and research labs
who have agreed to share their unused computational resources with each
other. If a job is submitted from an OSG submission point, it can run in

> the [Open Science Pool](https://osg-htc.org/about/open_science_pool/) associated with that submission point. This OSG pool is the

Author: Nope, this is from when Cannon purged Open Science Grid from the website four months ago.

running all over the campus or country, on computers that don\'t belong
to us, they have two major requirements:

-   **Moderate Data Sizes**: Ideally, all of the files for your jobs should be

>     in your home directory and/or in SQUID (the options listed on our [file

Author: yes, and my latest changes should have that information.

    transfer guide](file-availability) and our [SQUID
    guide](file-avail-squid)), and output files are small enough
    to return to your home directory (less than 2 GB).  If your data files 
    are large enough to use our large data file system ("staging", see our [data
    staging guide](file-avail-largedata)) and you would like to use external

>     resource, please [contact us](chtc@cs.wisc.edu)!  In certain cases, we may be 

Author: ```suggestion
    resources, please [contact us](chtc@cs.wisc.edu)!  In certain cases, we may be 
```
